lib/CMakeLists.txt

```diff
@@ -67,6 +67,12 @@ if(WIN32)
     list(APPEND VORBISFILE_SOURCES ../win32/vorbisfile.def)
 endif()
 
+if(MSVC)
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
```

Comment:
I was a bit nervous about this one. The "use our non-POSIX _s version" warnings are annoying, but `strcat` is usually a bad sign. The uses here are all safe though, so this is probably fine.